acpdr/swagger-specs/schema-registry.yaml

```diff
@@ -2071,7 +2201,81 @@ definitions:
             $ref:
               type: string
               example: '#/definitions/loyalty'
-  createMixinResponse:          
+  patchFieldGroupResponse:
+    type: object
+    example:
+      type: object
+      title: Loyalty Details
+      description: Field group to describe members of the Loyalty Program.
+      'meta:intendedToExtend':
+        - 'https://ns.adobe.com/xdm/context/profile'
+      definitions:
+        loyalty:
+          "properties":
+            "_{TENANT_ID}":
+              "type": "object"
+              "properties":
+                "loyaltyLevel":
+                  "title": "Loyalty Level"
+                  "description": "Member level within the loyalty program."
+                  "type": "string"
+                "memberSince":
+                  "title": "Member Since"
+                  "type": "string"
+                  "format": "date"
+                  "description": "Date loyalty member joined loyalty program."
```

Comment:
Feels a little weird to say "loyalty member joined loyalty program"? I'd go with just "Date member joined loyalty program."